src/Api/Serializer/MailSettingsSerializer.php

```diff
@@ -17,7 +17,7 @@ class MailSettingsSerializer extends AbstractSerializer
     /**
      * {@inheritdoc}
      */
-    protected $type = 'mail-settings';
+    protected $type = 'mail/settings';
```

Comment:
Was this change here before ? Doesn't it impact other code that has not been updated ?

Reply:
ack I thought I had removed that change, sorry. Shouldn't impact code, but definitely should be its own PR. Reverted,